package.json

```diff
@@ -13,7 +15,7 @@
     "react-dom": "^18.2.0",
     "react-redux": "^8.0.5",
     "react-router-dom": "^6.9.0",
-    "react-scripts": "5.0.1",
+    "react-scripts": "^5.0.1",
     "uuid": "^9.0.0",
     "web-vitals": "^2.1.4"
   },
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/230606556-b93f1d48-503f-4d70-99a6-a71b6cd7dbc8.png)

**[OPTIONAL]**

- [ ] Kindly fix this conflict because it can break your linter setup somehow and you might face some unexpected behaviour from your linters setup so kindly remove it and apologies because the previous reviewer didn't mention it 🤔